pong.py

```diff
@@ -17,10 +17,10 @@ def __init__(self, game, x, y):
         self.game.screen.fill("white", self.rect)
 
     def update(self):
-        if self.moving_up and self.rect.top < (self.game.screen_rect.top - 50):
+        if self.moving_up and self.rect.top > self.game.wall_top.bottom:
             self.y -= self.speed
             print("moving up")
-        if self.moving_down and self.rect.bottom > (self.game.screen_rect.bottom - 50):
+        if self.moving_down and self.rect.bottom < self.game.wall_bottom.top:
```

Comment:
this is the bug that was causing our thing to not work (the flipped comparison signs)